cypress/e2e/catalog/giftCards/updatingGiftCards.js

```diff
@@ -92,4 +94,57 @@ describe("As an admin I want to update gift card", () => {
         });
     },
   );
+
+  it(
+    "should be able to delete several gift cards. TC: SALEOR_1009",
+    { tags: ["@giftCard", "@allEnv", "@stable"] },
+    () => {
+      const name = `${startsWith}${faker.datatype.number()}`;
+      const name2 = `${startsWith}${faker.datatype.number()}`;
+
+      createGiftCard({
+        tag: name,
+        amount: 3,
+        currency: "THB",
+      })
+        .then(() => {
+          createGiftCard({
+            tag: name2,
+            amount: 7,
+            currency: "THB",
+          });
+        })
+        .then(() => {
+          cy.visit(urlList.giftCards)
+            .get(PRODUCTS_LIST.showFiltersButton)
+            .click()
+            .get(PRODUCTS_LIST.filters.filterBy.currency)
```

Comment:
I don't think that using filter by currency is a good way. Someone else can use it and how we will now that we are removing proper giftCards?